apple/Handlers/RNLongPressHandler.m

```diff
@@ -89,6 +92,7 @@ - (void)touchesMoved:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
   [_gestureHandler.pointerTracker touchesMoved:touches withEvent:event];
 
   if ([self shouldCancelGesture]) {
+    self.state = UIGestureRecognizerStateFailed;
```

Comment:
Is `UIGestureRecognizerStateFailed` intentionally being used instead of `UIGestureRecognizerStateCancelled` when gesture cancellation occurs?